src/modules/access-wallet/ModuleAccessWalletHardware.vue

```diff
@@ -97,7 +97,67 @@
           Ledger
         =====================================================================================
         -->
-      <span v-if="onLedger">Ledger</span>
+      <span v-if="onLedger">
+        <div class="subtitle-1 font-weight-bold mb-2">Connecting to:</div>
+        <div>
+          <mew-select
+            v-if="onLedger"
+            v-model="ledgerApp"
+            :items="ledgerApps"
+            :is-custom="true"
```

Comment:
only use this if you need icons and other various things in the dropdown! (look in storybook for more details or dm me)  sorry i don't have a ledger so i can't see it